tests/testRestGet.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Xunit;
```

Comment:
Depois me explica esse arquivo. To pensando se ele poderia ser menor

Reply:
esse é o arquivo de test do core kj